apps/src/applab/designElements/dropdown.jsx

```diff
@@ -88,6 +88,16 @@ module.exports = {
     element.style.height = '30px';
     element.style.fontSize = '14px';
     element.style.margin = '0';
+    element.style.color = '#fff';
+    element.style.backgroundColor = '#1abc9c';
```

Comment:
I wonder if we should have a list of colors we use somewhere, such that we can do something like require('./colors').green or something